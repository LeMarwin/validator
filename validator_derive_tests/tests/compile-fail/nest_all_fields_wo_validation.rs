use validator::Validate;

#[derive(Validate)]
#[validate(nest_all_fields)]
struct Test {
    nested: Nested,
    unchecked: u8,
}

#[derive(Validate)]
struct Nested {
    value: String,
}

fn main() {}
