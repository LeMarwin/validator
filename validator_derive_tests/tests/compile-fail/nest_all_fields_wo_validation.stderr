error[E0599]: the method `validate` exists for reference `&u8`, but its trait bounds were not satisfied
 --> tests/compile-fail/nest_all_fields_wo_validation.rs:3:10
  |
3 | #[derive(Validate)]
  |          ^^^^^^^^
  |
  = note: the following trait bounds were not satisfied:
          `u8: Validate`
          which is required by `&u8: Validate`
  = note: this error originates in the derive macro `Validate` (in Nightly builds, run with -Z macro-backtrace for more info)
